src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/VxlanFlowSpec.groovy

```diff
@@ -483,8 +483,9 @@ class VxlanFlowSpec extends HealthCheckSpecification {
 
         and: "Flow is pingable"
         verifyAll(northbound.pingFlow(flow.id, new PingInput())) {
-            it.forward.pingSuccess
-            it.reverse.pingSuccess
+            !forward
+            !reverse
+            error == "Flow ${flow.id} should not be one switch flow"
```

Comment:
I don't understand this. Just remove this ping verification because it is no longer valid. Why you should explicitly show that you are unable to ping 1-switch flow in every test, especially not ping-related?

Reply:
deleted in tests for APIv1